R/build_coxph.R

```diff
@@ -269,6 +481,36 @@ build_coxph.fast <- function(df,
       rf$terms_mapping <- names(name_map)
```

Comment:
Add todo to rename 'rf' to something more meaningful (probably 'model').

Reply:
Just replaced their names.